Add WKWebView for iOS for BCD (#30)
WebView-CG/Compatibility-Data-Project: _posts/2024-09-17-BCD-WKWebView.md

---
title:  "BCD Data now available for WKWebView 🎉"
date: 2024-09-17
---

> We're happy to announce that our friends in MDN have now added WKWebView to the [browser compatibility data](https://github.com/mdn/browser-compat-data)!

Reviewer: ```suggestion
We're happy to announce that our friends at [Open Web Docs](https://openwebdocs.org) have now added WKWebView to the [browser compatibility data](https://github.com/mdn/browser-compat-data)!
```
